platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/spi/ReadableKVStateBase.java

```diff
@@ -55,6 +53,17 @@ protected ReadableKVStateBase(@NonNull String stateKey) {
         this.stateKey = Objects.requireNonNull(stateKey);
```

Comment:
Use constructor chaining.

```suggestion
        this(stateKey, new ConcurrentHashMap<>());
```

Reply:
Thanks, comment addressed.